Algorithms.Test/PalindromeNumberTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Algorithms.Test
+{
+    public class PalindromeNumberTest
+    {
+        PalindromeNumber _palindromeNumber;
+        public PalindromeNumberTest() {
+            _palindromeNumber = new PalindromeNumber();
+        }
+        [Fact]
+        public void IsPalindrome_IsMoreOrEqualThan_ReturnFalse() {
+            int number = -2147483648;
+
+            var result = _palindromeNumber.IsPalindrome(number);
+            Assert.False(result);
+        }
+        [Fact]
+        public void IsPalindrome_IsLessOrEqualThan_ReturnFalse() { 
+            int number = 2147483647;
+
+            var result = _palindromeNumber.IsPalindrome(number);
+            Assert.False(result);
+        }
+        [Fact]
+        public void IsPalindrome_ValueZero_ReturnTrue() {
+            int number = 0;
+
+            var result = _palindromeNumber.IsPalindrome(number);
+            Assert.True(result);
+        }
+        [Fact]
+        public void IsPalindrome_NumberPositive_ReturnTrue() {
+            int number = 121;
+
+            var result = _palindromeNumber.IsPalindrome(number);
+            Assert.True(result);
+        }
+        [Fact]
+        public void IsPalindrome_NumberNegative_ReturnFalse()
```

Comment:
I'm guessing you used the one in leetcode for this exercise.
No need to do it, but how would you do it if for the problem didn't matter if it is negative or positive?
for example -515 should be true. What do you think you could do?